### PR TITLE
rearranged files and fixed imports

### PR DESCRIPTION
**Pull request by rymaju**
- Moved page specific files and components into src/pages/route, reusable components stay in /components

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,18 +2,17 @@ import React from 'react';
...
+import Apply from './pages/apply/Apply';
+import ApplyDeveloper from './pages/apply/ApplyDeveloper';
+import ApplyProductDesigner from './pages/apply/ApplyProductDesigner';
+import ApplyBrandDesigner from './pages/apply/ApplyBrandDesigner';
+import Projects from './pages/projects/Projects';
+import Jumpstart from './pages/jumpstart/Jumpstart';
+import Events from './pages/events/Events';
+import People from './pages/people/People';
+import FAQ from './pages/faq/FAQ';
+import NotFound from './pages/notfound/NotFound';
```
Comment (jackblanc) [inline src/App.tsx:15]: thoughts on renaming the files to `index.tsx`? 
That would allow `import Home from './pages/home'`

**Comment by rymaju** [comment]
- Changed css on Board.tsx (now People.tsx) removing complicated media queries and defaulting to mobile styling on all views.
- added scroll to top on home

**Comment by rymaju** [comment]
Its probably easier to just check nothing broke when I moved all those files around by checking out the amplify link.

File: src/pages/people/People.tsx [hunk 2]
```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Grid, Typography } from '@material-ui/core';
-import { ReactComponent as SVG } from '../svg/people.svg';
+import { ReactComponent as SVG } from '../../svg/people.svg';
```
Comment (jackblanc) [inline src/pages/people/People.tsx]: We could colocate SVGs to the /pages/people (for each) because we don't use any twice right?

**Review by jackblanc — COMMENTED** [review]
two suggestions

File: src/App.tsx (hunk 1 above)
Comment (rymaju) [reply src/App.tsx:15]: oo interesting, this also exposes a flaw in my directory structure: the ApplyDeveloper/Designer etc pages need to be in their own folder too

File: src/pages/people/People.tsx (hunk 2 above)
Comment (rymaju) [reply src/pages/people/People.tsx]: Very smart, I'll do that right now

File: public/sitemap.xml [hunk 3]
```diff
@@ -13,6 +13,9 @@
   </url>
   <url>
     <loc>https://c4cneu.com/apply/brand-designer</loc>
+  </url>
+    <url>
```
Comment (jackblanc) [inline public/sitemap.xml:17]: this looks not formatted

**Review by jackblanc — APPROVED** [review]

**Pull request merged**